railties/lib/rails/test_unit/reporter.rb

```diff
@@ -15,8 +15,12 @@ def aggregated_results # :nodoc:
       filtered_results.reject!(&:skipped?) unless options[:verbose]
       filtered_results.map do |result|
         location, line = result.method(result.name).source_location
-        "bin/rails test #{location}:#{line}"
+        "bin/rails test #{relative_path_for(location)}:#{line}"
       end.join "\n"
     end
+
+    def relative_path_for(file)
+      file.sub(/^#{Dir.pwd}\/?/, '')
```

Comment:
I ran the runner in a `--dev` app I have and saw failure output like:

```
bin/rails test /Users/kasperhansen/Documents/code/collection_caching_test/test/integration/cache_invalidation_test.rb:26
```

To me the path doesn't really have to be absolute and it would be more readable to just have them output relatively. This change turns the above into:

```
bin/rails test test/integration/cache_invalidation_test.rb:26
```

What do you think, @senny @arthurnn?